libs/portals/admin/form-system/src/components/FormLayout/components/MainColumns.tsx

```
@@ -1,22 +1,111 @@
-      <MainContent />
+      {containsGroupOrInput() ? (
+        <DialogPrompt
+          baseId="remove"
+          title={formatMessage(m.areYouSure)}
+          description={formatMessage(m.completelySure)}
+          ariaLabel="Remove item"
+          buttonTextConfirm={formatMessage(m.confirm)}
+          buttonTextCancel={formatMessage(m.cancel)}
+          onConfirm={remove}
+          disclosureElement={
+            <DeleteButton onClick={() => { }} label={formatMessage(m.delete)} />
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)